src/main/java/seedu/address/ui/ProjectDisplayPanel.java

```diff
@@ -121,7 +162,7 @@ public void showTodosTab() {
     /**
      * Custom {@code ListCell} that displays the graphics of an {@code Event} using an {@code EventCard}.
      */
-    class EventListViewCell extends ListCell<Event> {
+    static class EventListViewCell extends ListCell<Event> {
```

Comment:
Is it supposed to be < "Repeatable" > here

Reply:
Spotted this, left it because it was the original implementation. I'll go see if i can change this real quick.

Reply:
Ok. Not a big deal since we only have 1 type of repeatable

Reply:
Think this will take a little longer to fix (requires changing type of backing list & related methods). I suggest we push this to 1.4 and focus on 1.3 deliverables what do you think?